build/root/usr/local/bin/ci_entrypoint_gpu-operator.sh

```diff
@@ -46,6 +48,8 @@ collect_must_gather() {
 validate_gpu_operator_deployment() {
     trap collect_must_gather EXIT
 
+    toolbox/gpu-operator/set_repo-config.sh "${SCRIPT_DIR}"/ci_mirror.repo
```

Comment:
`build/root/usr/local/bin/ci_mirror.repo` isn't the right location for the repolist, maybe `build/resources`?

I would like to change (later, maybe when introducing the cli parsing framework) `build` to something more meaningful, maybe `ci_image`? (cc @ArangoGutierrez )